eks_fargate/README.md

```diff
@@ -250,6 +250,14 @@ spec:
 
 **Note**: Don't forget to replace `<YOUR_DATADOG_API_KEY>` with the [Datadog API key from your organization][13].
 
+### Live Containers
+
+Datadog Agent 6.19+ supports live containers in the EKS Fargate integration. Live containers appear in the [Containers][21] page.
+
+### Live Processes
+
+Datadog Agent 6.19+ supports live processes in the EKS Fargate integration. Live processes appear in the [Processes][22] page. To enable live processes, [enable `shareProcessNamespace` in the pod spec][23].
```

Comment:
should we provide a yaml with process agent and `shareProcessNamespace` enabled?  the same way we do for integrations, dogstatsd and traces